go.mod

```diff
@@ -1,5 +1,7 @@
 module go.linecorp.com/centraldogma
 
+go 1.19
```

Comment:
IMHO, centraldogma-go is backward-compatible with go1.17. How about just set it: go1.17 or go1.18